docs/_drafts/scv2-hodometer.md

```diff
@@ -0,0 +1,127 @@
+Aside from that, more frequent usage metrics aren't necessarily very useful.
+As the point is to provide an _indication_ of how Core v2 is being adopted and which features are (not) being used, this level of granularity is sufficient.
+Whether someone spins up a handful of extra inference servers for a few hours between Hodometer's runs or creates a new version of a pipeline before removing the old one doesn't fundamentally change the scale of adoption.
+Indeed, many of the metrics should be naturally slow-changing, such as the versions of Kubernetes and Core, or tend to oscillate within stable ranges, such as the number of servers deployed at any given time.
+I should note that this is based on assumptions and previous experience rather than hard data, but the nature of the problem means that precise data would be difficult and potentially controversial to attempt to collect.
+Sampled and/or aggregated data are always subject to missing data points and loss of detail.
+In any case, the collected metrics are generally further aggregated and assessed to identify _trends_, so inaccuracies in any given sample are highly unlikely to be significant in the grand scheme of things.
+
+## Making more mileage
+
+We've already touched upon the ability of end users to configure the metrics receiver endpoint.
+One additional point to note here is that Hodometer supports _extra_ receivers.
+This isn't something provided by Spartakus, but I decided to add it because it allows users to record the metrics whilst still providing them back to the maintainers.
+The aim is once again to inspire confidence through visibility whilst benefitting the community as a whole by sharing this information in an anonymised way.
+
+Another notable divergence from Spartakus is that Hodometer has the concept of _levels_ of metrics.
+While Spartakus treated it is an all-or-nothing situation, Core v2 gives the user the option to share only as much information as they're comfortable with.
+Making users select every individual metric would be tedious for them, so Hodometer groups these into levels: cluster-level, resource-level, and feature-level information.
+Each level is a superset of its predecessor.
+Cluster-level metrics are very basic: just the k8s and Core v2 scheduler versions (the scheduler is the heart of the control plane, so this implies the Core v2 version more generally).
+Resource-level metrics count the number of each Core v2-defined _custom resource_, such as the number of models and pipelines.
+It's important to note here that sensitive information like model names is _not_ recorded.
+Feature-level metrics are about how many servers have multi-model serving and over-committing enabled and how much memory servers have available to them.
+All of this can be seen in the `Collect` method [here](https://github.com/SeldonIO/seldon-core/blob/d3502062bbbb18a08032201917ceea07e124be41/hodometer/pkg/hodometer/collect.go#L227).
```

Comment:
:wrench: Should also link to `metrics.go` for the metrics definitions.  The `Collect` method best shows that levels are supersets/subsets of one another.